sr_gui_biotac/src/sr_gui_biotac/biotac.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) 2013, Shadow Robot Company, SynTouch LLC
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+#   * Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#   * Redistributions in binary form must reproduce the above
+#     copyright notice, this list of conditions and the following
+#     disclaimer in the documentation and/or other materials provided
+#     with the distribution.
+#   * Neither the name of the TU Darmstadt nor the names of its
+#     contributors may be used to endorse or promote products derived
+#     from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+# FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+
+import rospy
+import rospkg
+import os
+
+from qt_gui.plugin import Plugin
+from python_qt_binding import loadUi
+
+from QtGui import QMessageBox, QWidget, QIcon, QColor, QPainter, QFont
+from QtCore import QRectF, QTimer, SIGNAL, SLOT
+from sr_robot_msgs.msg import Biotac, BiotacAll
+
+NUMBER_OF_SENSING_ELECTRODES    = 19
+NUMBER_OF_EXCITATION_ELECTRODES = 4
+
+RECTANGLE_WIDTH  = 45
```

Comment:
all those below can probably be kept as CONSTS, although the names are not too descriptive (why 1 2 3 4 and not array?)